pallets/valve/src/traits.rs

```diff
@@ -7,3 +7,11 @@ pub trait Shutdown {
 	/// Forwards restart message to implementer
 	fn restart();
 }
+
+impl Shutdown for () {
+	fn is_shutdown() -> bool {
```

Comment:
what does this mean?

Reply:
this is because we ripped out automationPrice pallet. Valve expects automation pallet and automation time pallets to be passed into it so that it can shut down the pallet as needed. In this case, since we removed automation price, there's nothing to pass into Valve. Therefore, we have to pass a "mock" item into valve. What valve is expecting is an object with a trait `Shutdown`, as defined above. A [similar implementation](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/pallets/automation-price/src/lib.rs#L880) was made in automation price. Here, rather than defining a mock automationPallet, we just used `()` and passed that into Valve instead to stub the missing pallet